numpy/random/tests/test_random.py

```diff
@@ -167,6 +167,14 @@ def test_random_integers(self):
                             [-48, -66]])
         np.testing.assert_array_equal(actual, desired)
 
+    def test_random_integers_max_int(self):
```

Comment:
The test looks iffy. Since we are just testing if the maximum is obtainable and the range is closed, maybe make both `high` and `low` equal to `np.iinfo('l').max`. Also needs a comment as to what is being tested for.